Detectors/MUON/MCH/Workflow/src/DataDecoderSpec.cxx

```diff
@@ -152,8 +160,9 @@ class DataDecoderTask
     std::ifstream in(filename);
     while (std::getline(in, s)) {
       content += s;
+      content += " ";
     }
-    return s;
+    return content;
```

Comment:
given the number of bugs I've introduced in this class, we should really think about proving a unit test for it ;-)

Reply:
Actually, I expect the custom mapping to be useful only in the early stages of the development and testing, and only for very specific cases, so a unit test might be over-doing...